honeybee_doe2/properties/model.py

```diff
@@ -96,14 +96,32 @@ def _make_mats_cons_layers(obj):
         return ConstructionCollection.from_hb_constructions(constructions=cons).to_inp()
 
     @property
-    def hvac_sys_zones(self):
-        return self._get_hvac_sys_zones(self.stories)
+    def hvac_sys_zones_by_model(self):
+        return self._get_hvac_sys_by_model(self.host)
 
     @staticmethod
-    def _get_hvac_sys_zones(stories):
+    def _get_hvac_sys_by_model(obj):
+        hvac_sys = [HVACSystem.from_model(obj)]
+        return hvac_sys
+
+    @property
+    def hvac_sys_zones_by_story(self):
+        return self._get_hvac_sys_zones_by_story(self.stories)
+
+    @staticmethod
+    def _get_hvac_sys_zones_by_story(stories):
         hvac_sys_zones = [HVACSystem.from_story(story) for story in stories]
         return hvac_sys_zones
 
+    @property
+    def hvac_sys_zones_by_room(self):
+        return self._get_hvac_sys_zones_by_room(self.host)
+
+    @staticmethod
+    def _get_hvac_sys_zones_by_room(obj):
+        hvac_sys_zones = [HVACSystem.from_room(room) for room in obj.rooms]
+        return hvac_sys_zones
+
```

Comment:
Not a big fan of adding many `staticmethod`s. Why don't you just use the property in this case?

```suggestion
    def _get_hvac_sys_by_model(obj):
        hvac_sys = [HVACSystem.from_model(obj)]
        return hvac_sys

    @property
    def hvac_sys_zones_by_story(self):
        hvac_sys_zones = [HVACSystem.from_story(story) for story in self.stories]
        return hvac_sys_zones

    @property
    def hvac_sys_zones_by_room(self):
        hvac_sys_zones = [HVACSystem.from_room(room) for room in self.host.rooms]
        return hvac_sys_zones

```

I would also add some docstrings here for your future self! You will thank yourself for that! 😄

Reply:
done! [see commit](https://github.com/ladybug-tools/honeybee-doe2/pull/59/commits/d5c09e8a92e4ee07d881212f93fd1e19257f584d)